generators/sql-constants.js

```diff
@@ -25,7 +25,7 @@ const databaseData = {
     protocolSuffix: 'sqlserver://',
     port: ':1433;database=',
     r2dbc: {
-      protocolSuffix: 'mssql://',
+      protocolSuffix: 'mariadb://',
```

Comment:
Are you sure you made this change in the right spot? This seems to be for SQL Server.